package.json

```diff
@@ -29,9 +29,7 @@
     "ethereumjs-tx": "^1.3.4",
     "ethereumjs-util": "^5.1.5",
     "mongodb": "^3.1.0-beta4",
-    "web3": "^0.20.1",
-    "web3-latest": "npm:web3",
-    "yarn": "^1.7.0"
+    "web3": "1.0.0-beta.34"
```

Comment:
Is web3 1.0.0+ ready for production? I have issues in the past with this library and had intentionally overwritten to maintain v0.20.0+

I would love to move to 1.0.0 for things like signing libraries that i currently use ethutils for.

Reply:
I often run into the issue or others that require yarn to install web3 i think 

https://github.com/ethereum/web3.js/issues/950

Reply:
Double version of web3 removed, if still issues we can re-instate yarn.